third_party/abseil-cpp/absl/synchronization/mutex.cc

```diff
@@ -59,6 +59,49 @@
 #include "absl/synchronization/internal/per_thread_sem.h"
 #include "absl/time/time.h"
 
+#ifndef _WIN32
+#include <dlfcn.h>
+#endif
+
+static void* LookupRecordReplaySymbol(const char* name) {
+#ifndef _WIN32
+  void* fnptr = dlsym(RTLD_DEFAULT, name);
+#else
+  HMODULE module = GetModuleHandleA("windows-recordreplay.dll");
+  void* fnptr = module ? (void*)GetProcAddress(module, name) : nullptr;
+#endif
+  return fnptr ? fnptr : reinterpret_cast<void*>(1);
+}
+
+static void RecordReplayBeginDisallowEventsWithLabel(const char* label) {
```

Comment:
I think I asked this before: Why is this `RecordReplayBeginDisallowEventsWithLabel` instead of `recordreplay::BeginDisallowEventsWithLabel`?

Reply:
This avoids link errors, abseil-cpp is a more general purpose library used in different scenarios by chrome.